spec/factories/moods.rb

```diff
@@ -2,7 +2,7 @@
   factory :mood do
     association :user
 
-    mood { [1, 2, 3, 4, 5].sample }
+    mood { [0, 1, 2, 3, 4].sample }
```

Comment:
Front end will be sending back 1-5 as mood type (I think), can we change this to reflect that?

Front end chime in whenever you review this.

Reply:
Talking to Natalia she said 0-4 is fine